app/views/rentals/index.html.erb

```diff
@@ -16,9 +16,12 @@
   <%= f.label :item_type_id_eq, 'Item Type' %>
   <%= f.select :item_type_id_eq, options_from_collection_for_select(@item_types.order(name: :asc), "id", "name", selected: (params[:q].try(:[], :item_type_id_eq) || 0)), {include_blank: true}, class: "selectpicker form-control col-xs-2" %>
 
+  <%= f.label :item_id_eq, 'Item Name' %>
+  <%= f.select :item_id_eq, options_from_collection_for_select(@items.order(name: :asc), "id", "name", selected: (params[:q].try(:[], :item_id_eq) || 0)), {include_blank: true}, class: "selectpicker form-control col-xs-2" %>
+
   <%= f.label :user_id_eq, 'Customer' %>
   <%= f.select :user_id_eq, options_from_collection_for_select(@users.order(first_name: :asc), "id", "full_name", selected: (params[:q].try(:[], :user_id_eq) || 0)), {include_blank: true}, class: "selectpicker form-control col-xs-2" %>
-
+<br><br>
```

Comment:
Would you like to take a stab at this CSS before moving on (hint hint: say yes!)